package.json

```diff
@@ -42,12 +42,12 @@
     "eslint-plugin-react": "^7.23.2",
     "fs-extra": "^9.1.0",
     "glob": "^7.1.4",
-    "got": "^11.8.1",
-    "html2plaintext": "^2.1.2",
+    "html-to-text": "^8.0.0",
```

Comment:
It seemed like @alexander-akait chose to move these dependencies from the individual packages out into the workspace in cssnano/cssnano/pull/804. I assumed that it was to make maintenance/keeping them up to date simpler by managing it all in one place? Perhaps I misunderstood though; I'm not that familiar with the lerna universe yet

Reply:
Interesting. I think at the time the repository did not use workspaces, so devDependencies would end up duplicated in every sub-package. If you put package-specific dependencies the workspace, when you run a command in that single package it might fail because it cannot find the dependency. For example right now, you cannot re-build a single package at a time because `yarn` cannot find `rimraf` since it is declared in the workspace (for example `yarn workspace stylehacks run build`).  I guess it makes little practical difference in this case, since these are test dependencies and you cannot run the tests from inside a single package anyway.

Reply:
> I guess it makes little practical difference in this case, since these are test dependencies and you cannot run the tests from inside a single package anyway.

Indeed, and the tool/script itself is also generally only useful in the context of maintaining the repo. Off the top of my head, I can't really think of any likely use cases where it would be used by itself. As it's not actually packaged/distributed with `postcss-reduce-initial`. But that perhaps also raises the question if these kind of scripts would be better suited living in `/util` together with the build and test helpers rather than under the package. Perhaps also something to think about if we circle back to the question of organisation at some point. 🤔 

Come to think of it....

> it's not actually packaged/distributed with `postcss-reduce-initial`

... the ignore pattern in the build command for this package appears to be something I have overlooked to update.